RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+Pin graphql-core to only accept patch versions.
```

Comment:
```suggestion
This releases pins graphql-core to only accept patch versions in order to prevent breaking changes since graphql-core doesn't properly follow semantic versioning.
```